src/PhpSpec/Process/ReRunner/WindowsPassthruReRunner.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Process\ReRunner;
+
+use PhpSpec\Process\Context\ExecutionContextInterface;
+use Symfony\Component\Process\PhpExecutableFinder;
+
+class WindowsPassthruReRunner extends PhpExecutableReRunner
+{
+    /**
+     * @var ExecutionContextInterface
+     */
+    private $executionContext;
+
+    /**
+     * @param PhpExecutableFinder $phpExecutableFinder
+     * @param ExecutionContextInterface $executionContext
+     * @return static
+     */
+    public static function withExecutionContext(PhpExecutableFinder $phpExecutableFinder, ExecutionContextInterface $executionContext)
+    {
+        $reRunner = new static($phpExecutableFinder);
+        $reRunner->executionContext = $executionContext;
+
+        return $reRunner;
+    }
+
+    /**
+     * @return boolean
+     */
+    public function isSupported()
+    {
+        return (php_sapi_name() == 'cli')
+            && $this->getExecutablePath()
+            && function_exists('passthru')
+            && (stripos(PHP_OS, "win") === 0);
+    }
+
+    public function reRunSuite()
+    {
+        $args = $_SERVER['argv'];
+        $command = $this->buildArgString() . '"' . $this->getExecutablePath().'" '.join(' ', array_map('escapeshellarg', $args));
```

Comment:
The quote marks either side of the call to the executable sort this out, though I am working on the assumption that the output of the getExecutablePath() method is 'clean' and doesn't have anything within it that needs escaping itself. The PHP escape functions cause the command not to run under windows for some reason. My PHP executable is in Program Files and this change works with it.